src/app/producer-shelf/components/tv2-action-card/tv2-action-card.component.ts

```diff
@@ -1,16 +1,60 @@
-import { ChangeDetectionStrategy, Component, HostBinding, Input } from '@angular/core'
-import { Tv2Action } from '../../../shared/models/tv2-action'
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, ElementRef, HostBinding, Input, OnInit, ViewChild } from '@angular/core'
+import { Tv2Action, Tv2ActionContentType, Tv2VideoClipAction } from '../../../shared/models/tv2-action'
+import { ConfigurationService } from '../../../shared/services/configuration.service'
+import { StudioConfiguration } from '../../../shared/models/studio-configuration'
+
+const ASPECT_RATIO: number = 1.77777777
```

Comment:
To make it easier for others to know what the ratio is, the expression could be changed to the ratio fraction `const ASPECT_RATIO: number = 16 / 9`.